packages/components/src/popover/stories/index.story.tsx

```diff
@@ -255,3 +258,76 @@ export const WithSlotOutsideIframe: StoryObj< typeof Popover > = {
 		...Default.args,
 	},
 };
+
+export const WithCloseHandlers: StoryObj< typeof Popover > = {
+	render: ( args ) => {
+		const WithHandlers = () => {
```

Comment:
No need to have the extra internal `WithHandlers` component.

If you're doing this to avoid React Hook errors, you can just use a named function for the `render` prop instead:

```suggestion
	render: function WithCloseHandlersStory( args ) {
```

Reply:
Will do so, thanks!